content/02.author-sections.md

```diff
@@ -589,6 +587,17 @@ This is an improved version of my main figure in this interesting study [@doi:10
 
 
 <!-- Rashika Ramola -->
+### Rashika Ramola
+Hi I am Rashika Ramola. I am a PhD student at Northeastern University. This is my first PSB. I like computational biology and I am excited to be here.
+
+[My first paper] (https://psb.stanford.edu/psb-online/proceedings/psb19/ramola.pdf) studies some performance measures (accuracy, balanced accuracy, f-measure and Matthews Correlation Coefficient) in positive-unlabeled learning. In this work, we demonstrate how performance measure can be inaccurate in positive unlabeled setting, and then we introduce correction measures.
```

Comment:
If you want to cite by the doi instead of the URL you can use `My first paper [@doi:10.1142/9789813279827_0012]` in place of the link.